spec/factories/vacancies.rb

```diff
@@ -250,5 +228,26 @@
       school_offer { nil }
       flexi_working { nil }
     end
+
+    factory :draft_vacancy do
```

Comment:
Question: Shouldn't this follow the same "class:..." as `factory :vacancy, class: "PublishedVacancy" do`

```suggestion
    factory :draft_vacancy, class: "DraftVacancy" do
```